test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -42,7 +42,9 @@ echo "Running framework tests."
 
 
 if [ $FRAMEWORK = "tensorflow" ]; then
-    timeout 1800s pytest mock/tests/${FRAMEWORK} -k "not test_mnist_s3 and not test_mnist_s3_multi_save_configs" --ignore=mock/tests/mxnet/test_training_end.py  2>&1
+    timeout 1800s pytest mock/tests/${FRAMEWORK} \
```

Comment:
whats the change here?

Reply:
The `test_keras` test function has been added to the tests that must be skipped, using the "-k" option.

Reply:
is this meant to be done for both tf1 and tf2 or just tf2? i think as of now this will only affect tf1

Reply:
this is meant for only tf1. for tf-2 branch smdebug doesn't have this test .